guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc

```diff
@@ -12,7 +12,7 @@ For more information, see xref:Creating_a_Custom_File_Type_Repository_{context}[
 . The `katello-server-ca.crt`.
 For more information, see {AdministeringDocURL}sect-Administering-Installing_the_Katello_Root_CA_Certificate[Installing the Katello Root CA Certificate] in the _Administering {ProjectName}_ guide.
 . An Organization Debug Certificate.
-For more information, see xref:Creating_an_Organization_Debug_Certificate_{context}[].
+For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificate_managing-organizations-locations[].
```

Comment:
I haven't tested this but it has an impact on Satellite?

Reply:
Yes it does, that's why I've overwritten the attribute for Satellite builds. Then it should work normally. Can you test this locally?

See `git grep -i -n "ManagingOrganizationsLocationsDocURL"`.

Reply:
This is how it renders when I build the Satellite scenario: 



        An Organization Debug Certificate. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html-single/content_management_guide/index#Creating_an_Organization_Debug_Certificate_managing-organizations-locations.

I fear we will need to do ifdefs.

Reply:
ah, now I see. We need ifdefs due to the "hardcoded" context, which would be `content-management` for satellite, and else `managing-organizations-locations`.

Reply:
We've had this before. We reuse between the installation guides, so installing foreman and smart proxy has reuse, as well as upstream and downstream contexts... there should be a way around this other than ifdefs. I think setting the context in master to a variable. I will look but tomorrow at this stage.

Reply:
I feel that we can implement a solution if there were more areas impacted than this, and I'm happy to merge at this point. 
The webpage will need to be updated at some point with a link to the new guide.